import_export/resources.py

```diff
@@ -452,19 +452,19 @@ def after_import(self, dataset, result, using_transactions, dry_run, **kwargs):
         """
         pass
 
-    def before_import_row(self, row, **kwargs):
+    def before_import_row(self, row, row_number=None, **kwargs):
         """
         Override to add additional logic. Does nothing by default.
         """
         pass
 
-    def after_import_row(self, row, row_result, **kwargs):
+    def after_import_row(self, row, row_result, row_number=None, **kwargs):
```

Comment:
Couldn't `row_number` be a a `kwarg` here? Why do we need to add it explicitly?

Reply:
I wanted to get the reference to the autogenerated documentation, otherwise the change would be quite invisible.

Reply:
@andrewgy8 Do you have any other solution to get this properly documented?